.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: rust
+cache: cargo
+rust:
+    - 1.35.0
+    - stable
+    - beta
+    - nightly
+os:
+    - linux
+    - osx
+matrix:
+    allow_failures:
+        - rust: nightly
+
+before_script:
+    - |
+      (travis_wait rustup component add rustfmt-preview || true) &&
+      (test "$TRAVIS_RUST_VERSION" == 1.35.0 || travis_wait rustup component add clippy-preview || true)
```

Comment:
I'd suggest removing these conditions and just doing the right-hand side, like `travis_wait rustup ...` ‒ that was in the original because the 1.24 didn't have clippy or rust-fmt yet.

Reply:
Right, addressed in fcc2d6634d80879785c7027e478d77f70f8db915